platform/android/java/lib/build.gradle

```diff
@@ -104,7 +104,7 @@ android {
         }
 
         boolean devBuild = buildType == "dev"
-        boolean debugSymbols = devBuild || isAndroidStudio()
+        boolean debugSymbols = devBuild
```

Comment:
Not necessarily for this PR, but I think the back and forth on this shows that we need a dedicated option for enabling debug symbols in gradle, and not just infer it from `dev` or debug symbols.

It should be possible for users to make `template_release` and `template_debug` builds with debug symbols, to be stripped when uploading their builds, so they can symbolicate crashes.